webtreemap.js

```diff
@@ -12,11 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+;(function(root, factory) {
+  if (typeof define === 'function' && define.amd)
+    define([], factory);
+  else if (typeof module === 'object' && module.exports)
+    module.exports = factory();
+  else
+    root.appendTreemap = factory();
+}(this, function() {
```

Comment:
`this` is window in the base browser context, so the else branch means you still get window.appendTreemap.

Better comments are here: https://github.com/umdjs/umd/blob/master/templates/returnExports.js